package.json

```
@@ -64,13 +75,13 @@
     "url": "https://github.com/digitalbazaar/http-client/issues"
   },
   "homepage": "https://github.com/digitalbazaar/http-client",
-  "module": "main.js",
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=14.14.0"
```

Is this minor version correct? If not, we need to determine what it actually needs to be or just do this:

```suggestion
    "node": ">=14"
```

---

This was suggested by @davidlehn. 

@davidlehn can you confirm if this is correct

---

https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c says both 14.14 and ^14.13.1.  ky-universal says >=14.  ky says >=12.  Who knows.  >=14 is fine and will work good enough I imagine.

---

Okay, I'll update it to >=14 then

---

Updated [74d2acd](https://github.com/digitalbazaar/http-client/pull/21/commits/74d2acd2e1f27d778ef0f4684dcf20683f9c3742)